javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -143,6 +147,11 @@ public static ResettableClassFileTransformer installBytebuddyAgent(
     agentBuilder = customizeByteBuddyAgent(agentBuilder);
     log.debug("Installed {} instrumenter(s)", numInstrumenters);
     ResettableClassFileTransformer resettableClassFileTransformer = agentBuilder.installOn(inst);
+    try {
+      Class.forName("io.opentelemetry.javaagent.instrumentation.api.concurrent.RunnableWrapper");
```

Comment:
What is this? :)

Reply:
that was part of today's deadlock work, it's gone now